src/qt/guiutil.cpp

```diff
@@ -610,6 +610,9 @@ void openConfigfile()
 {
     fs::path pathConfig = GetConfigFile(gArgs.GetArg("-conf", BITCOIN_CONF_FILENAME));
 
+    /* Create the file */
+    fs::ofstream configFile(pathConfig, std::ios_base::app);
+
```

Comment:
(13982) I'm not sure if this is correct. It certainly isn't what upstream did. I seem to think we probably have done another backport that makes this one irrelevant, but I'm not sure

Reply:
This is a Dash-specific code implemented in #181. A similar one was introduced in bitcoin 9890 2 years later but we skipped that PR for some reason. In any case, this change is unrelated and should be removed.